src/main/scala/com/exasol/cloudetl/kafka/KafkaConsumerProperties.scala

```diff
@@ -101,6 +101,12 @@ class KafkaConsumerProperties(private val properties: Map[String, String])
   final def getTableName(): String =
     getString(TABLE_NAME)
 
+  /**
+   * Checks if user wants to consume all offsets of topic partitions.
```

Comment:
Maybe also mention here that returns a default false if not set by the user?

Reply:
Done.